dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -97,18 +103,24 @@ protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
      */
     protected boolean isGithubAppEnabledOnRepositoryWithGitApi(String fullRepoName) {
         refreshJwtIfNeeded(appId, privateKeyPath);
-        try {
-            gitHub.getApp().getInstallationByRepository(fullRepoName.split("/")[0], fullRepoName.split("/")[1]);
-            return true;
-        } catch (HttpException exception) {
-            if (exception.getResponseCode() != 404) {
-                // Log for any HTTP status code other than 404 Not found. 
-                log.warn("Caught a HTTPException {} while trying to get app installation. Defaulting to False", exception.getMessage());
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            String apiEndpoint = "https://git.soma.salesforce.com/api/v3/repos/" + fullRepoName + "/installation";
+            HttpGet httpGet = new HttpGet(apiEndpoint);
+            httpGet.setHeader("Authorization", jwt);
+            httpGet.setHeader("Accept", "application/vnd.github+json");
+
+            HttpResponse response = httpClient.execute(httpGet);
+            int statusCode = response.getStatusLine().getStatusCode();
+            if (statusCode == 200) {
+                return true;
+            } else if (statusCode == 404) {
+                return false;
+            } else {
+                log.warn("[isGithubAppEnabledOnRepositoryWithGitApi] -- Unexpected response code `{}` while trying to get app installation. Defaulting to False.", statusCode);
+                return false;
```

Comment:
We can log the status code in all situations and then simplify this if else ladder to `return statusCode == 200`

Reply:
@cjneasbi Done